### PR TITLE
[ws_v7.x.x] Add definitions

### PR DESCRIPTION
**Pull request by eidheim**
- Links to documentation: https://github.com/websockets/ws/blob/master/doc/ws.md
- Link to GitHub or NPM: https://github.com/websockets/ws
- Type of contribution: new definition

Other notes:
This adds definition for one of the most popular JS WebSocket libraries.

### CODE REVIEW

File: definitions/npm/ws_v7.x.x/flow_v0.104.x-/test_v7.x.x.js [hunk 1]
```diff
@@ -0,0 +1,382 @@
+// @flow
+
```
Comment (AndrewSouthpaw) [inline definitions/npm/ws_v7.x.x/flow_v0.104.x-/test_v7.x.x.js:2]: Please use `describe` and `it` blocks to organize the test file a little more.

**Review by AndrewSouthpaw — CHANGES_REQUESTED** [review]
Looks great, thanks for that work! One change requested...

File: definitions/npm/ws_v7.x.x/flow_v0.104.x-/test_v7.x.x.js (hunk 1 above)
Comment (eidheim) [reply definitions/npm/ws_v7.x.x/flow_v0.104.x-/test_v7.x.x.js]: Thank you for the review. I organised the tests in the commit below.

**Review by AndrewSouthpaw — APPROVED** [review]
Thanks for contributing a new libdef!

**Pull request merged**